interpreter/k8s/src/main/scala/io/buoyant/interpreter/k8s/IstioInterpreter.scala

```diff
@@ -2,14 +2,17 @@ package io.buoyant.interpreter.k8s
 
 import com.twitter.finagle._
 import com.twitter.finagle.naming.NameInterpreter
+import io.buoyant.k8s.SingleNsNamer
 import io.buoyant.k8s.istio.RouteCache
 import io.buoyant.namer.ConfiguredDtabNamer
 import istio.proxy.v1.config.RouteRule
 
 object IstioInterpreter {
   private val istioPfx = "/#/io.l5d.k8s.istio"
+  private val k8sPfx = "/#/io.l5d.k8s.ns"
   private val defaultRouteDtab = Dtab.read(s"""
     |/svc/ext => /$$/inet ;
```

Comment:
based on my updated understanding of egress, I think we should delete this dentry.  `/svc/ext` names are generated when the host header doesn't match any clusters (even "external" ones).  These requests should fail.